setup.cfg

```diff
@@ -3,9 +3,9 @@ name = amsterdam-schema-tools
 version = 4.1.0
 url = https://github.com/amsterdam/schema-tools
 license = Mozilla Public 2.0
-author = Amsterdam Data en Informatie
+author = Team Data Diensten of the Dataplatform under the Directie Digitale Voorzieningen (Gemeente Amsterdam)
```

Comment:
Een beetje nengels. Niet gewoon compleet nederlands doen, of de namen tussen quotes?